ISSUE_TEMPLATE.md

```diff
@@ -3,3 +3,4 @@
 * * * *
 
 - [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
+- [ ] I am an instance admin who has checked that I'm running a [tagged release](https://github.com/tootsuite/mastodon/releases) and _not_ the latest master.
```

Comment:
:+1: on @wxcafe's suggestion, we don't want to necessarily discourage people from reporting issues on master, but we do want to know if it *is* on master, because issues in tagged releases are more dire